components/brave_shields/resources/panel/shields_panel.tsx

```diff
@@ -38,6 +38,7 @@ function App () {
     const onVisibilityChange = () => {
       if (document.visibilityState === 'visible') {
         getPanelBrowserAPI().panelHandler.showUI()
+        getPanelBrowserAPI().dataHandler.updateFavicon()
```

Comment:
Curious only needs explicit favicon update?

Reply:
yes, internally i'm sending favicon url as part of `SiteBlockInfo` struct - on an active tab it will send the right url but sometimes resources take time to load so `OnFavIconUpdate()` it will internally update the url by cache busting. 

However, when opening the panel on a new tab it doesn't trigger an update again so this `updateFavicon` helps do that. maybe I should rename the function if that's confusing?